<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>500</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="MySubwayView" name="subwayView">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>出行目的</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>起点</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QComboBox" name="startLine"/>
             </item>
             <item>
              <widget class="QComboBox" name="startStation"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>终点</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QComboBox" name="endLine"/>
             </item>
             <item>
              <widget class="QComboBox" name="endStation"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>出行策略</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QRadioButton" name="minDistanceBtn">
              <property name="text">
               <string>最短距离</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="minTransferBtn">
              <property name="text">
               <string>最少换乘</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="minStationBtn">
              <property name="text">
               <string>最少站点</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="confirmBtn">
              <property name="text">
               <string>确定</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="exitAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="addAction"/>
    <addaction name="separator"/>
    <addaction name="zoomoutAction"/>
    <addaction name="separator"/>
    <addaction name="zoominAction"/>
    <addaction name="separator"/>
    <addaction name="recoverAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="helpAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="aboutMakerAction"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="exitAction"/>
   <addaction name="addAction"/>
   <addaction name="zoomoutAction"/>
   <addaction name="zoominAction"/>
   <addaction name="recoverAction"/>
   <addaction name="helpAction"/>
   <addaction name="aboutMakerAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/icon/exit.jpeg</normaloff>:/icon/icon/exit.jpeg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>退出程序</string>
   </property>
  </action>
  <action name="addAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/icon/add.jpeg</normaloff>:/icon/icon/add.jpeg</iconset>
   </property>
   <property name="text">
    <string>动态添加</string>
   </property>
   <property name="toolTip">
    <string>动态添加线路</string>
   </property>
  </action>
  <action name="zoomoutAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/icon/zoomOut.jpg</normaloff>:/icon/icon/zoomOut.jpg</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大视图</string>
   </property>
  </action>
  <action name="zoominAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/icon/zoomIn.jpg</normaloff>:/icon/icon/zoomIn.jpg</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小视图</string>
   </property>
  </action>
  <action name="recoverAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/icon/graph.jpeg</normaloff>:/icon/icon/graph.jpeg</iconset>
   </property>
   <property name="text">
    <string>恢复视图</string>
   </property>
  </action>
  <action name="helpAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/icon/help.jpeg</normaloff>:/icon/icon/help.jpeg</iconset>
   </property>
   <property name="text">
    <string>使用帮助</string>
   </property>
  </action>
  <action name="aboutMakerAction">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/icon/about.jpeg</normaloff>:/icon/icon/about.jpeg</iconset>
   </property>
   <property name="text">
    <string>制作者</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MySubwayView</class>
   <extends>QGraphicsView</extends>
   <header>mysubwayview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
